docs/cli.adoc

```diff
@@ -306,6 +307,7 @@ ____
 To lint three files, writing machine-readable messages to _/tmp/vim-qflist.json_:
 ____
 [subs=+quotes]
+[source,bash,subs=+quotes,role="scrollable-container"]
```

Comment:
Instead of applying this fix to specific items, could we apply it site-wide? Other pages are affected, such as https://quick-lint-js.com/contribute/create-diagnostic/#report.

Reply:
I removed `[source,bash,subs=+quotes,role="scrollable-container"]` from the items and added to main.css:
```css
@media screen and (max-width: 670px) {
  pre, code {
    overflow-x: scroll;
    max-height: 400px;
  }
}
```

However i cannot figure out an solution for the scroll bar to always be visible, maybe that would also be browser dependent too.

I do agree on your comment that it's better to have content which is obviously discoverable rather than fancy content which is harder to discover, so given the minor nature of this issue, implementing such a solution may be a bit of an overkill.